src/Teams.Web/Controllers/ManageSprintsController.cs

```diff
@@ -89,6 +97,28 @@ public async Task<IActionResult> GetSprintById(int sprintId)
             return View(sprint);
         }
 
+        [Authorize]
+        public async Task<IActionResult> CompleteTaskInSprint(int taskId, bool isCompleted)
+        {
+            // Представление не передает данные об объекте
+            //task.Completed = isCompleted;
+
+            var task = await _manageTasksService.GetTaskByIdAsync(taskId);
+            
+            task.Completed = isCompleted;
```

Comment:
why this behaviour in controller?

Reply:
where logic to check is sprint assigned or not?
right now incorrect logic that sprint is required
also need to check that sprint isActive (will be changed when it will be Status instead of isActive flag, but anyway should be Active)